FirebaseSegmentation/Tests/Sample/SegmentationSampleApp.xcodeproj/project.pbxproj

```diff
@@ -392,7 +386,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.google.firebase.config.testapp.dev;
+				PRODUCT_BUNDLE_IDENTIFIER = com.google.firebase.paul.hackweek.nov2020;
```

Comment:
I'm ok with leaving this after hackweek, but wanted to drop a comment on it in case this bundle id change was unintentional

Reply:
Thanks. Longer term, it should be updated to something that runs correctly and is shared for any developers of this SDK. For now, it builds again and in CI.